state/src/impls/top_level.rs

```diff
@@ -618,533 +553,6 @@ impl TopState for TopLevelState {
         Ok(())
     }
 }
-
-#[cfg(test)]
```

Comment:
I think we should not naively remove some tests. Though the tests are related to state changes due to Account-related transactions, `clone`, `override_cache`, etc are closely related to general db operations.

Reply:
If some suitable transactions are not available from the top_level, let's apply module level state changes and examine the same tests onto it.